source/index.html.md

```diff
@@ -67,175 +67,394 @@ Kittn expects for the API key to be included in all API requests to the server i
+            "scm": {
+                "price": 1301546,
+                "volume": 5
+            },
+            "item_name": "★ Karambit | Doppler",
+            "wear_name": "Factory New",
+            "phase": "Ruby",
+            "description": "It has been painted with black and silver metallic paints using a marbleizing medium, then candy coated.\\n\\n<i>Getting lost in its color can prove fatal</i>",
+            "badges": []
+        },
+        "is_seller": false,
+        "min_offer_price": 675750,
+        "max_offer_discount": 1500,
+        "is_watchlisted": false,
+        "watchers": 0
+    }
 ]
 ```
 
-This endpoint retrieves all kittens.
+This endpoint retrieves all listings.
 
 ### HTTP Request
 
-`GET http://example.com/api/kittens`
+`GET https://csgofloat.com/api/v1/listings`
 
 ### Query Parameters
 
 Parameter | Default | Description
 --------- | ------- | -----------
-include_cats | false | If set to true, the result will also include cats.
-available | true | If set to false, the result will include kittens that have already been adopted.
-
-<aside class="success">
-Remember — a happy kitten is an authenticated kitten!
-</aside>
-
-## Get a Specific Kitten
-
-```ruby
-require 'kittn'
-
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.get(2)
-```
-
-```python
-import kittn
-
-api = kittn.authorize('meowmeowmeow')
-api.kittens.get(2)
-```
+page | 0 | Which page of listings to start from
+limit | 100 | How many listings to return
+order | `best_deal` | How to order the listings
+category | 0 | Can be one of: 0 = `any`, 1 = `normal`, 2 = `stattrak`, 3 = `souvenir`
+def_index | | Only include listings that have a one of the given def index(es)
+min_float | | Only include listings that have a float higher then this
+max_float | | Only include listings that have a float lower then this
+rarity | | Only include listings that have this rarity
+paint_seed | | Only include listings that have this paint seed
+paint_index | | Only include listings that have this paint index
+user_id | | Only include listings from this user
+collection | | Only include listings from this collection
+min_price | | Only include listings have a price higher then this
+max_price | | Only include listings have a price lower then this
+market_hash_name | | Only include listings that have this market hash name
+type | | Either `buy_now` or `auction`
+stickers | | Must be in the form: `<STICKER-ID>\|<POSITION>[,<STICKER-ID>\|<POSITION>...]`
+
+## Get a Specific Listing
 
 ```shell
-curl "http://example.com/api/kittens/2" \
-  -H "Authorization: meowmeowmeow"
-```
-
-```javascript
-const kittn = require('kittn');
-
-let api = kittn.authorize('meowmeowmeow');
-let max = api.kittens.get(2);
+curl "https://csgofloat.com/api/v1/listings/<LISTING-ID>"
 ```
 
 > The above command returns JSON structured like this:
 
 ```json
 {
-  "id": 2,
-  "name": "Max",
-  "breed": "unknown",
-  "fluffiness": 5,
-  "cuteness": 10
+    "id": "292312870132253796",
```

Comment:
Similar stuff here with obfuscation and pruning. Should probably use a listing by a developer (maybe mine here: https://csgofloat.com/item/324288155723370196) to not publicize user details more than we have to.